static/js/dxr.js

```diff
@@ -1,553 +0,0 @@
-        jsm: "js",
-        py: "py",
-        ini: "conf",
-        sh: "sh",
-        txt: "txt",
-        xml: "xml",
-        xul: "ui",
-        java: "java",
-        in: "txt",
-        html: "html",
-        png: "image",
-        gif: "image",
-        svg: "svg",
-        build: "build",
-        json: "js",
-        css: "css",
-      }[suffix] || "unknown"
-    );
-  }
-
-  function renderPath(pathkind, qkind, fileResult) {
-    var klass = classOfResult(pathkind, qkind);
-
-    var html = "";
-    html += "<tr class='result-head " + klass + "'>";
-    html +=
-      "<td class='left-column'><div class='" +
-      chooseIcon(fileResult.path) +
-      " icon-container'></div></td>";
-
-    html += "<td>";
-
-    var elts = fileResult.path.split("/");
-    var pathSoFar = "";
-    for (var i = 0; i < elts.length; i++) {
-      if (i != 0) {
-        html += "<span class='path-separator'>/</span>";
-      }
-
-      var elt = elts[i];
-      pathSoFar += elt;
-      html += "<a href='" + makeURL(pathSoFar) + "'>" + elt + "</a>";
-      pathSoFar += "/";
-    }
-
-    html += "</td>";
-    html += "</tr>";
-
-    return html;
-  }
-
-  function renderSingleSearchResult(pathkind, qkind, file, line) {
-    var [start, end] = line.bounds || [0, 0];
-    var before = line.line.slice(0, start).replace(/^\s+/, "");
-    var middle = line.line.slice(start, end);
-    var after = line.line.slice(end).replace(/\s+$/, "");
-
-    var klass = classOfResult(pathkind, qkind);
-    var html = "";
-    html += "<tr class='" + klass + "'>";
-    html +=
-      "<td class='left-column'><a href='" +
-      makeURL(file.path) +
-      "#" +
-      line.lno +
-      "'>" +
-      line.lno +
-      "</a></td>";
-    html += "<td><a href='" + makeURL(file.path) + "#" + line.lno + "'>";
-
-    function escape(s) {
-      return s.replace(/&/gm, "&amp;").replace(/</gm, "&lt;");
-    }
-
-    html += "<code>";
-    html += escape(before);
-    html += "<b>" + escape(middle) + "</b>";
-    html += escape(after);
-    html += "</code>";
-
-    html += "</a>";
-
-    if (line.context) {
-      var inside = line.context;
-      if (line.contextsym) {
-        var url = `/${Dxr.tree}/search?q=symbol:${encodeURIComponent(
-          line.contextsym
-        )}&redirect=false`;
-        inside = "<a href='" + url + "'>" + line.context + "</a>";
-      }
-      html +=
-        " <span class='result-context'>// found in <code>" +
-        inside +
-        "</code></span>";
-    }
-
-    html += "</td>";
-    html += "</tr>";
-
-    return html;
-  }
-
-  var count = 0;
-  for (var pathkind in data) {
-    for (var qkind in data[pathkind]) {
-      for (var k = 0; k < data[pathkind][qkind].length; k++) {
-        var path = data[pathkind][qkind][k];
-        count += path.lines.length;
-      }
-    }
-  }
-
-  var fileCount = 0;
-  for (var pathkind in data) {
-    for (var qkind in data[pathkind]) {
-      fileCount += data[pathkind][qkind].length;
-    }
-  }
```

Comment:
Why did you remove this? This will completely break searchfox lol.

Reply:
oups, I went full in :)
I will rename this script to make it obvious that it isn't dxr.